### PR TITLE
computing relative attention bias

### PR DESCRIPTION
**Pull request by pmabbo13**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #1829
* #1828
* #1827
* #1826
* #1833
* #1832
* __->__ #1831
* #1830
* #1825

### CODE REVIEW

**Review by Nayef211 — APPROVED** [review]
LGTM

**Comment by pmabbo13** [comment]
# Description

This module computes the relative position embeddings. This is the bias term that will be used when computing the attention scores. Our implementation is taken from [HF](https://github.com/huggingface/transformers/blob/8581a798c0a48fca07b29ce2ca2ef55adcae8c7e/src/transformers/models/t5/modeling_t5.py#L421).

File: torchtext/prototype/t5/modules.py [hunk 1]
```diff
@@ -54,6 +54,33 @@ def __init__(
...
+    def _compute_bias(
+        self,
+        query_length: int,
+        key_length: int,
+        relative_attention_bias: Tensor,
+        relative_attention_num_buckets: int = 32,
+        relative_attention_max_distance: int = 128,
+        bidirectional: bool = True,
+        device=None,
+    ):
```
Comment (parmeet) [inline torchtext/prototype/t5/modules.py:67]: nit: Annotate return type

**Review by parmeet — APPROVED** [review]
LGTM!

**Pull request merged**